docs/tutorials/configure/pet-photographer.md

```diff
@@ -0,0 +1,263 @@
+---
+title: "Pet Photographer: Filter Data Captures"
+linkTitle: "Pet Photographer"
+type: "docs"
+description: "Use the filter modular component in the Viam app to photograph your pet in their collar."
+tags: ["vision", "filter", "camera", "detector", "services"]
+aliases:
+  - /tutorials/pet-photographer
+  - /tutorials/filter-modular-component
+authors: ["Sky Leilani"]
+languages: []
+viamresources: ["vision", "camera"]
+level: "Beginner"
+date: "2023-09-17"
+# updated: ""
+cost: "0"
+no_list: true
+weight: 3
+---
+
+Smart machines are an integral part of our daily lives.
+From our phones to traffic lights, these machines rely on their ability to process and respond to data.
+However, as the amount of data collected by these devices continues to grow, it becomes important to not only process it, but also to organize and manage it effectively.
+
+This tutorial will guide you through using the color filter module to selectively capture and synchronize image data for pet photography with [Viam's cloud](/services/data/#cloud-sync).
+You can follow these steps to enhance your data management whether you're configuring a camera to take pictures of your pet, or working with any robot equipped with a [camera](/components/camera/)component.
+
+While the color filter module you use in this tutorial selects image data from a camera, these same principles can be applied to various components, including for filtering [sensor](https://github.com/viam-labs/modular-filter-examples/tree/main/sensorfilter) data.
+The filter modular component allows you to [store data](/services/data/#data-capture) only when that data meets certain conditions you set.
```

Comment:
Sorry - I know I've already asked you to change this once but I think this is better yet
```suggestion
The filter modular component allows you to selecively [store data](/services/data/#data-capture) when that data meets certain conditions you set.
```